dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/maatapi/MaatApiWebClientFactory.java

```diff
@@ -51,7 +51,7 @@ public WebClient maatApiWebClient(
                 HttpClient.create(provider)
                     .resolver(DefaultAddressResolverGroup.INSTANCE)
                     .compress(true)
-                    .responseTimeout(Duration.ofSeconds(30))
+                    .responseTimeout(Duration.ofSeconds(90))
```

Comment:
Increase the read timeout for maatapi clients to 90 seconds as 30 seconds was insufficient for integration tests (on development env)